src/item.cpp

```diff
@@ -4318,20 +4326,10 @@ int item::get_encumber_when_containing(
             break;
     }
 
-    const int thickness = get_thickness();
-    const int coverage = get_coverage();
-    if( item_tags.count( "wooled" ) ) {
-        encumber += 1 + 3 * coverage / 100;
-    }
-    if( item_tags.count( "furred" ) ) {
-        encumber += 1 + 4 * coverage / 100;
-    }
-
-    if( item_tags.count( "leather_padded" ) ) {
-        encumber += ceil( 2 * thickness * coverage / 100.0f );
-    }
-    if( item_tags.count( "kevlar_padded" ) ) {
-        encumber += ceil( 2 * thickness * coverage / 100.0f );
+    for( auto &cm : clothing_mods::get_all() ) {
```

Comment:
This at a minimum need to do something like ``clothing_mods::get_all_with( "warmth" )`` otherwise the scaling can go out of control. This is a current issue with effects, so it's not hypothetical.
Ideally it would just iterate over the "present" effects only and retrieve their impacts, but I can see that might be very difficult.

Reply:
I pushed using cache trial version.